src/DeltaLake/Kernel/State/ParquetStringPartitions.cs

```diff
@@ -0,0 +1,35 @@
+// -----------------------------------------------------------------------------
+// <summary>
+// The list of parquet partitions.
+// </summary>
+//
+// <copyright company="The Delta Lake Project Authors">
+// Copyright (2024) The Delta Lake Project Authors.  All rights reserved.
+// Licensed under the Apache license. See LICENSE file in the project root for full license information.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace DeltaLake.Kernel.State
+{
+    /// <summary>
+    /// The list of parquet string partitions.
+    /// </summary>
+    internal struct ParquetStringPartitions
+    {
+        /// <summary>
+        /// The number of partitions.
+        /// </summary>
+        public int Len;
+
+        /// <summary>
+        /// The list of partition column names.
+        /// </summary>
+        public unsafe char** ColNames;
```

Comment:
It's as much pedantry as anything else because these never get dereferenced, but as these are both UTF8 strings they'd be more correctly described as `byte**` instead of `char**`.

Reply:
Done, thank you for the tip :)